Source/WebGPU/WebGPU/ShaderModule.mm

```diff
@@ -80,6 +80,36 @@
     return library;
 }
 
+// FIXME: https://bugs.webkit.org/show_bug.cgi?id=247030 - (Remove placeholder test shader)
+static String fallbackMSLString()
+{
+    return String::fromUTF8(
```

Comment:
Can we not actually commit this? How harmful would it be to maintain this in a local checkout until we get something better?

Reply:
Certainly not harmful, just nothing will show on the screen until the compiler outputs a basic shader like this one